.github/workflows/branch-sync.yml

```diff
@@ -135,7 +135,7 @@ jobs:
         run: |
           if [[ -n "$RUNNER_DEBUG" ]]; then set -x; fi
 
-          git switch "$SYNC_BRANCH"
+          git switch -c "$SYNC_BRANCH" "origin/${HEAD_BRANCH}"
```

Comment:
Partially reverts #120 - I think I got confused by the failures at the time and missed the fact that this step should only run if the sync branch does not exist yet

Reply:
No probs.